### ISSUE TITLE
Uploads cancelled right after being created on macOS

### ISSUE THREAD

**Issue by crcunningham** [tus/TUSKit#75]
Looking at this commit: https://github.com/tus/TUSKit/commit/aad200876a80a8b48b4dcb366195290a58d73d8e

I see this code a few times: 

```
       #if TARGET_OS_IPHONE
            [[UIApplication sharedApplication] endBackgroundTask:bgTask];
        #elif TARGET_OS_MAC
            [weakself cancel];
        #endif
```

It looks like this is some sort of copy+paste error. Can you confirm?

**Comment by MMasterson** [comment]
In the current master branch? Hm, yeah - let me take a look

**Comment by MMasterson** [comment]
closed in #76
